src/ucs/config/parser.h

```diff
@@ -91,6 +91,7 @@ typedef struct ucs_config_global_list_entry {
     ucs_config_field_t       *table;   /* array of configuration fields */
     size_t                   size;     /* size of config structure */
     ucs_list_link_t          list;     /* entry in global list */
+    int                      loaded; /* entry was loaded by parser */
```

Comment:
Align comment

Reply:
WDYT of making a `flags` member instead of dedicating a whole `int` to a single bit?
Then can be extended if need extra fields in the future